src/main/java/frc/robot/commands/FaceReef.java

```diff
@@ -0,0 +1,91 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import java.util.function.Supplier;
+
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Constants;
+import frc.robot.Constants.DriveConstants;
+import frc.robot.Constants.ReefConstants;
+import frc.robot.subsystems.DriveSubsystem;
+import frc.robot.utilities.FaceReefUtil;
+import frc.robot.utilities.PIDF;
+import frc.robot.utilities.TunablePIDF;
+
+/* You should consider using the more terse Command factories API instead https://docs.wpilib.org/en/stable/docs/software/commandbased/organizing-command-based.html#defining-commands */
+public class FaceReef extends Command {
+  DriveSubsystem m_drive;
+  Supplier<Double> m_xVelocitySupplier;
+  Supplier<Double> m_yVelocitySupplier;
+  FaceReefUtil m_reef;
+
+  private static final TunablePIDF targetTurningPIDF = new TunablePIDF("Reef.turningPIDF", ReefConstants.kTurningPIDF);
+
+  private ProfiledPIDController m_thetaController = new ProfiledPIDController(
+    targetTurningPIDF.get().p(),
+    targetTurningPIDF.get().i(),
+    targetTurningPIDF.get().d(),
+    new TrapezoidProfile.Constraints(
+      DriveConstants.kMaxAngularSpeedRadiansPerSecond,
+      DriveConstants.kMaxAngularAccelerationRadiansPerSecondSquared),
+    Constants.kDt);
+
+  /** Creates a new TargetReef. */
```

Comment:
s/TargetReef/FaceReef/, or just remove since the comment borders on tautological.